libs/mark-flow-ui/src/pages/cast_ballot_page.tsx

```diff
@@ -0,0 +1,85 @@
+/* stylelint-disable order/properties-order */
+import React from 'react';
+import styled from 'styled-components';
+
+import {
+  Button,
+  Font,
+  H1,
+  Icons,
+  InsertBallotImage,
+  Main,
+  P,
+  Screen,
+  VerifyBallotImage,
+} from '@votingworks/ui';
+
+const Instructions = styled.ol`
+  margin: 2rem 0;
+  padding: 0;
+`;
+
+const ListItem = styled.li`
+  align-items: center;
+  display: flex;
+  font-weight: ${(p) => p.theme.sizes.fontWeight.semiBold};
+  gap: 1rem;
+  margin-bottom: 1.5rem;
+`;
+
+const InstructionImageContainer = styled.div`
+  border-right: ${(p) => p.theme.sizes.bordersRem.thick}rem solid
+    ${(p) => p.theme.colors.foreground};
+  display: flex;
+  justify-content: center;
+  padding: 1rem 1rem 0 0;
+  width: 30vw;
+`;
+
+const Done = styled.div`
+  position: absolute;
+  right: 1rem;
+  bottom: 1rem;
+`;
+
+interface Props {
+  hidePostVotingInstructions: () => void;
+}
+
+export function CastBallotPage({
```

Comment:
Was this meant to be a file move? (Didn't see a corresponding delete) Or will the original copy continue to be used in VxMark?

Reply:
This one should not have been moved or copied as the corresponding RAVE one will be different.